R/process_sc_all_alarms_telecare.R

```diff
@@ -16,16 +16,14 @@ process_sc_all_alarms_telecare <- function(
     sc_demog_lookup,
     write_to_disk = TRUE) {
   # Data Cleaning-----------------------------------------------------
-
   replaced_dates <- data %>%
-    # period start and end dates
-    dplyr::mutate(
-      record_date = end_fy_quarter(.data$period),
-      qtr_start = start_fy_quarter(.data$period)
-    ) %>%
+    dplyr::mutate(service_end_date = fix_sc_missing_end_dates(
+      .data$service_end_date,
+      .data$period_end_date
+    )) %>%
```

Comment:
```suggestion
    dplyr::mutate(service_end_date = fix_sc_missing_end_dates( .data$service_end_date, .data$period_end_date), 
      service_start_date = fix_sc_start_dates(.data$service_start_date, .data$period_start_date)
    ) %>%
```
Logic looks fine/agree with it. Once suggestion to wrap the mutate for creating the variables as it simplifies the code